src/server/connections.rs

```diff
@@ -55,12 +62,31 @@ async fn connection_loop(co: &mut Connection) {
             tx,
         });
     }
+    let metric_connected = METRIC_MOLLYSOCKET_SIGNAL_CONNECTED.with_label_values(&[
+        &co.strategy.clone().to_string(),
+        &co.uuid.clone(),
+        &co.endpoint.clone(),
+    ]);
+    let metric_push = METRIC_MOLLYSOCKET_PUSH.with_label_values(&[
```

Comment:
i do not understand - what do you mean with the `CONFIG` part.
the parameter is generated here with the label values.
so it could not be a global.

Inside of `src/ws/signalwebsocket.rs` there i would miss the `co.uuid` maybe we could parse that over (with `co.device_id` and `&co.password` and generate it there (together with `CONFIG.get_ws_endpoint(&co.uuid, co.device_id, &co.password)` )